proto/storage/virtual_machine.proto

```diff
@@ -25,15 +25,18 @@ message VirtualMachine {
     MISSING_SIGNATURE_VERIFICATION_DATA = 3;
   }
   repeated Note notes = 9;
+
+  int32 vsock_cid = 10;
+  bool running = 11;
```

Comment:
I don't know about the precise rationale for this field (consider this comment non-blocking), but wouldn't be safer to use an enum here? bool forces us to set it to true or false, and there is usually a situation where we want to say "unknown" or similarly to pod status there may be "starting", "failed" etc. Enum would cover for that and would be future-proof. Wdyt?

Reply:
Excellent suggestion. Will adjust accordingly.